boring/src/ssl/mod.rs

```diff
@@ -2240,6 +2240,16 @@ impl ClientHello<'_> {
     pub fn version_str(&self) -> &'static str {
         self.ssl().version_str()
     }
+
+    /// Returns the client hello raw data
+    pub fn raw_client_hello(&self) -> &[u8] {
```

Comment:
This is the entire client hello record, right? Then I would just name that as `as_bytes`.

Reply:
Yes that's the whole message sent by the client. Agreed that's a better name, will change.